packages/interface/src/config.ts

```diff
@@ -115,6 +115,15 @@ export const eas = {
   },
 };
 
+export const zupass = {
+  pcdType: "eddsa-ticket-pcd",
+  publicKey: [
+    "1ebfb986fbac5113f8e2c72286fe9362f8e7d211dbc68227a468d7b919e75003",
```

Comment:
are these constants for all eddsa-ticket-pcd?

Reply:
Currently, they are, in theory at some point they would change who could sign pcds.

Reply:
oh ok